app/controllers/employment/CheckYourBenefitsController.scala

```diff
@@ -112,9 +114,14 @@ class CheckYourBenefitsController @Inject()(implicit val appConfig: AppConfig,
           case Some(cya) =>
             employmentSessionService.createModelOrReturnResult(cya, prior, taxYear, EmploymentSection.EMPLOYMENT_BENEFITS) match {
               case Left(result) => Future.successful(result)
-              case Right(model) => employmentSessionService.createOrUpdateEmploymentResult(taxYear, model).flatMap {
+              case Right(model) =>
+                employmentSessionService.createOrUpdateEmploymentResult(taxYear, model).flatMap {
                 case Left(result) => Future.successful(result)
                 case Right(returnedEmploymentId) =>
+                  checkYourBenefitsService.performSubmitAudits(model, employmentId, taxYear, prior)
+                  if (appConfig.nrsEnabled) {
```

Comment:
Is this NRS stuff part of the ticket?

Reply:
can't see why it shouldn't be added in here, nrs submission is there already for details, expenses etc

Reply:
Confirmed it is with Ben